assets/js/components/ReportTable.stories.js

```diff
@@ -136,12 +136,18 @@ export default {
 	component: ReportTable,
 	decorators: [
 		( Story, { parameters } ) => {
+			const setupRegistry = ( registry ) => {
+				if ( parameters?.setupRegistry ) {
+					parameters.setupRegistry( registry );
+				}
+			};
+
 			return (
-				<WithTestRegistry features={ parameters.features || [] }>
+				<WithRegistrySetup func={ setupRegistry }>
 					<Layout>
-						<Story />
+						<Story { ...parameters } />
```

Comment:
`parameters` are not the same as args. These were used before to pass through features but features are already handled at the top level in preview. I'm also not sure we even need to pass through new props to `Story` since these should already be included. (I don't think it is the same as passing to the template).